rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/rm/ListNodeCommand.java

```diff
@@ -76,8 +75,18 @@ public void execute(ApplicationContext currentContext) throws CLIException {
                     selectedNodeEvents = selectedList.toArray(new NodeEventView[selectedList.size()]);
                 }
             }
-            resultStack(currentContext).push(selectedNodeEvents);
-            writeLine(currentContext, "%s", StringUtility.string(selectedNodeEvents));
+            List<NodeEventView> filtered = new ArrayList<>(selectedNodeEvents.length);
+            for (NodeEventView nodeEvent : selectedNodeEvents) {
+                if (!"NODE_REMOVED".equalsIgnoreCase(nodeEvent.getEventType())) {
+                    filtered.add(nodeEvent);
+                } else {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)